research/conditional/utils/model_utils.py

```diff
@@ -535,13 +538,21 @@ def get_classes_from_module_names(
             classes.append(ExpertGating)
         elif name == "Softmax":
             classes.append(torch.nn.Softmax)
```

Comment:
I def think this should be reverted (see below also)

Reply:
Now I don't think so :)

Reply:
sly